task.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+  "id": "2b1ae9cd-0a4f-4059-b563-61ae159f7496",
+  "name": "DownloadMultipleSecureFiles",
+  "friendlyName": "Download multiple secure files",
+  "description": "Download multiple secure files to the agent machine",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Adrian Ochmann",
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 0
+  },
+  "visibility": ["Build", "Release"],
+  "runsOn": ["Agent", "DeploymentGroup"],
+  "instanceNameFormat": "Download multiple secure files",
+  "inputs": [
+    {
+      "name": "secureFiles",
+      "type": "multiLine",
+      "label": "Secure Files",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "The file name or GUID of the secure files to download to the agent machine. The files will be deleted after the pipeline runs."
```

Comment:
Needs more description/help. Eg. "each filename needs to be placed in the new line". Remember about plurals: "fileS", "GUIDS" etc.

Reply:
Will be done in: https://github.com/aochmann/azure-download-multiple-secure-files/pull/2